tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+pushd OpenTripPlanner
+mvn test -Dgpg.skip -Dmaven.javadoc.skip=true
```

Comment:
Does the onebusaway compile step also need to run?

Reply:
Yes, it's still work in progress.